src/AppInstallerCommonCore/Archive.cpp

```diff
@@ -1,6 +1,13 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 #include "pch.h"
+#include "winget/Archive.h"
+#pragma warning( push )
+#pragma warning ( disable : 4189 )
+#pragma warning ( disable : 4244 )
+#pragma warning ( disable : 26451 )
+#include "pure.h"

```

Comment:
pure.h should probably go to pch.h

Reply:
Discussed offline, leaving this as is. Added a todo statement to move this to pch.h in the future.